src/tests/alttestproj/compose.yaml.golden

```diff
@@ -0,0 +1,33 @@
+name: test
+services:
+  dfnx:
+    build:
+      context: ../../tests/alttestproj
+      dockerfile: Dockerfile
+      args:
+        DNS: dfnx
+      target: alttestproj
+    deploy:
+      resources:
+        limits:
+          cpus: 0.5
+          memory: "536870912"
+        reservations:
+          cpus: 0.25
+          memory: "268435456"
```

Comment:
any reason why memory is a string?

Reply:
guess they use a string representation internally?